docs/docs/bandits/config.mdx

```diff
@@ -0,0 +1,90 @@
+The Decision Metric is the metric that the Bandit will optimize towards. After each **reweighting** update, the Bandit will assign more traffic to the variations that are performing better on the Decision Metric. It should be a metric that you care about improving and that you can measure quickly after the user is exposed to the experiment.
+
+An ideal decision metric has the following properties:
+
+- It has a **short conversion window** and can be observed in a relatively short time frame. Short conversion windows allow the bandit to update with accurate information; a metric that takes longer to be observed will slow the Bandit down.
+- If it is a proportion metric, the conversion rate should not be super low or super high. These extreme conversion rates can result in Bandits that are very slow to converge as a lot of data is needed to adequately separate the Bandit variations.
+
+<img
+  src="/images/bandits/bandit-config-2.png"
+  alt="Configuring Bandit Decision Metric and Exploration Window"
+  style={{ width: 600, margin: "0 auto" }}
+/>
+
+### 4. Set the exploration window and update cadence
+
+The Bandit needs to know how long it should collect data for before beginning to update variation weights. This is called the **Exploration Window**. After the Exploration Window, the Bandit will begin to update variation weights at a set **Update Cadence**.
+
+**Exploration Window**
+
+Your Exploration Window should be long enough that you get enough traffic into each arm before the Bandit begins optimizing traffic. We _require_ at least 100 users per variation before we make any updates to our Bandit.
+
+You should make your Exploration Window long enough to:
+
+- Get a good number of conversions or metric values to begin estimating your Decision Metric. We recommend having at least 40 conversions per variation before you begin updating your Bandit. If your conversion rate tends to be around 10%, then we recommend picking a window long enough to get 400 units per variation.
+- Get a decent mix of different units into your experiment (e.g. running a 8 hour Exploration Window may only capture users in particular time zones before it begins updating).
+
+**Update Cadence**
+
+The minimum update cadence you can set in GrowthBook Bandits is 15 minutes. However, because each update requires tracking events to flow from your users to your data warehouse, and then into GrowthBook via queries, in practice your data may take longer to materialize and the practical minimum update may be longer.
+
+Furthermore, the fewer units that enter your Bandit in each period, the less valuable it is to update the Bandit frequently.
+
+Unless you have a compelling reason to update your Bandit more frequently, we recommend setting your update cadence to daily or even longer, depending on the goals of your bandit. Of course, longer update windows mean that the Bandit will not be able to adapt as quickly to lower traffic to losing variations, but it can be more stable.
+
```

Comment:
We should also ask them to check their SDK and endpoint/CDN caching settings. Specifically, they want to make sure the SDK Cache uses a relatively short `maxAge` (or `TTL`) so that weight changes go into effect quickly (otherwise we slow down results and introduce some chance for SRM). Our default settings on most of our SDKs are reasonable.